core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/validation/ValidationIssueDetails.java

```diff
@@ -0,0 +1,25 @@
+package com.b2international.snowowl.core.validation;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map.Entry;
+
+import com.b2international.snowowl.core.ComponentIdentifier;
+
+public class ValidationIssueDetails {
+	
+	public static final String HIGHLIGHT_DETAILS = "highlightDetails";
+	public final List<Entry<Integer, Integer>> detailEntries;
+	public final ComponentIdentifier affectedComponentId;
+
+	public ValidationIssueDetails(ComponentIdentifier affectedComponentId) {
+		this.detailEntries = Collections.emptyList();
+		this.affectedComponentId = affectedComponentId;
+	}
+	
+	public ValidationIssueDetails(List<Entry<Integer, Integer>> detailEntries, ComponentIdentifier affectedComponentId) {
```

Comment:
A `List<Entry<Integer, Integer>>` is essentially a `Map` if the list should not contain any duplicate entry, which in this case is true, I think. Also, rename the `detailEntries` field to something meaningful.
Please refactor.